javascript-timers/timeout.js

```diff
@@ -0,0 +1,3 @@
+var timeOutID = window.setTimeout(function () {
+  document.getElementById('message').innerHTML = "Hello there";
```

Comment:
Directly setting `innerHTML` is dangerous and slow, please use `textContent` instead.

Reply:
Got it